envs/environment.yaml

```
@@ -44,7 +44,7 @@ dependencies:
 - pydoe2
 - shapely!=2.0.4
 - pre-commit
-- pyomo
+- scip!=9.2.0
```

I think it may be a good idea to keep also a comment on the dependency. What do you think?

```suggestion
- scip!=9.2.0  # dependency of pyscipopt, temporary fix
```